Field.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace gameoflife;
+
+class Field
+{
+    protected $field = [];
+    protected $puffer = [];
+    protected $generationCount = 1;
+    protected $width;
+    protected $height;
+    protected $maxSteps;
+    protected $history = [];
```

Comment:
Why is this protected? Is it needed by subclasses? If not, it should be private!